src/parser.rs

```diff
@@ -72,16 +75,20 @@ named!(fname<String>, alt!(quoted | bare));
  * Header lines
  */
 
-named!(header_line_content<String>,
+named!(header_line_content<(String, Option<DateTime<FixedOffset>>)>,
     chain!(
         filename: fname ~
-        take_until!("\n") ,
+        opt!(space) ~
```

Comment:
[`space()`](http://rust.unhandledexpression.com/nom/fn.space.html) "Recognizes spaces and tabs".

I prefer skipping WS here rather than (maybe not) skipping it inside `after`, as separation of concerns.